modules/http/src/main/java/org/mule/module/http/api/requester/proxy/NtlmProxyConfigBuilder.java

```diff
@@ -78,4 +78,14 @@ public NtlmProxyConfigBuilder setNtlmDomain(String ntlmDomain)
         return this;
     }
 
+    /**
+     * @param nonProxyHosts A list of hosts separated by |, which specifies that the proxy must not be used against those
```

Comment:
Just realised, Mule uses `,` for this scenarios. For example, the status code validator.